bw2io/importers/json_ld.py

```diff
@@ -132,7 +132,7 @@ def boolcheck(lst):
             {
                 "code": obj["@id"],
                 "name": obj["name"],
-                "categories": category_mapping[obj["category"]["@id"]],
+                "categories": category_mapping[obj["category"]["@id"]], #There is no 'category'... does he mean 'type'?
```

Comment:
What does the [olca-schema](https://pypi.org/project/olca-schema/) say about this ?